tools/vmd/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+exports_files(["ios_vm_test_runner.template.sh"])
+
+sh_binary(
+    name = "vmd",
+    srcs = ["vm_run.sh"],
+    data = [
+        # Note: there are a plethora of problems from doing a trival build of this
+        # depenency with Bazel. For now, vendor the binary (56kb)
+        #
+        # We could always make a certificate - this needs further auditing for
+        # production usage.
+        "bin/sshpass",
+        "@tart//file",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+sh_binary(
+    name = "ios_vm_test_runner",
+    srcs = ["ios_vm_test_runner.sh"],
+    data = [
+        ## We want to call the original runner under vm_run
+        "vmd",
+        "@xctestrunner//:ios_test_runner",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+# Wrap everything into a CLI
+pkg_tar(
+    name = "vmd_tar",
```

Comment:
Maybe add an example usage of this? I don't see if being referenced anywhere, I understand the intent but just so others have a starting point to understand what is being packed here and how it's supposed to be used

Reply:
You can see this if you run `bazel run tools/vmd`:
https://github.com/bazel-ios/rules_ios/pull/509/commits/af562b5a60395ea05bc5538e8c27c7bd39eb8eac#diff-7d31e8bdec909f1ff9a767b67732bb653cff61404ab816e92d2e6c6d9e0a3007R4